### DISCUSSION TITLE
Persisting Github Action Error

### DISCUSSION THREAD

**Discussion by johko** [huggingface/computer-vision-course#357] [Announcements]
Hey folks :wave: 

thanks for doing all the hard work of creating new content and fixing broken stuff.

## The Problem
Unfortunately for quite some time now there is an error that occurs within a GitHub action (upload-pr-documentation).

I have been investigating this quite a lot now. Here is a quick (3 point) rundown:
1. When a PR is created, an updated version of the whole course is uploaded to a HF Hub repo
2. From this repo, a preview of the whole course is created, to check for any possible formatting issues
3. Unless this action has gone through, we cannot publish to the main site

## The Solution
After a lot of investigation my best guess is, that something is wrong with the token needed to push to this repo.
Unfortunately, I cannot fix it on my own as I'm lacking the authority to create new access tokens with the needed permissions on the Hugging Face Hub. This means someone with the authority (people from the core HF team) needs to create a new token for me to test.

So far none of my efforts to reach someone from the team have been successful, they all seem to be quite busy recently. I'm very sorry for the inconvenience and hope to get a solution soon, so we can get all your amazing content out there :muscle:

**Comment by adamm-hf** [comment]
Thanks for investigating! Forwarding to the team 🤗

**Reply by Roman-Malinowski** [reply]
Hi, any news from HF team? 🤗

**Reply by johko** [reply]
Yes, we are working on solving the error together now. Hopefully it will soon be done.
